tests/regression/success/frame_block_no_init.lus

```diff
@@ -0,0 +1,21 @@
+node main(c1, c2: bool) returns (out: int);
+var x1, x2 : int;
+let
+  frame (x1, x2)
+  let
+    if c1
+    then
+      x1 = 24;
+    else 
+      if c2
+      then
+        x1 = 37;
+      else
+        x1 = 34;
+      fi
+    fi
+  tel
+
+  check (true -> (x1 = 24 or x1 = 37 or x1 = 34));
```

Comment:
I would replace this check with this stronger version:
`check (x1 = 24 or x1 = 37 or x1 = 34);`